JenkinsfileDeploy

```diff
@@ -0,0 +1,69 @@
+    def userInput
+
+    stage('Set properties') {
+        timeout(time: 5, unit: 'MINUTES') {
+            userInput = input(
+                id: 'select-deployment',
+                message: 'Select environment & service',
+                parameters : [
+                    string(name: 'targetRemoteHost', description: 'Hostname where to deploy the subgraph', defaultValue: '', trim: true),
+                    string(name: 'networkName', description: 'Network Name', defaultValue: 'bellecour', trim: true),
+                    string(name: 'voucherHubAddress', description: 'Voucher Hub Address', defaultValue: '0x3137B6DF4f36D338b82260eDBB2E7bab034AFEda', trim: true),
+                    string(name: 'voucherHubStartBlock', description: 'Start Block for Voucher Hub', defaultValue: '30306387', trim: true),
+                    string(name: 'versionLabel', description: 'Version Label for Subgraph Deployment', defaultValue: 'develop', trim: true),
+                    string(name: 'dockerImageVersion', description: 'Version of the deployer docker image to use', defaultValue: '', trim: true)
+                ]
+            )
+        }
+        println """
+            Target remote host env: $userInput.targetRemoteHost
+            Graph Node URL: http://$userInput.targetRemoteHost:8020
+            IPFS URL: http://$userInput.targetRemoteHost:5001
+            Network Name: $userInput.networkName
+            Voucher Hub Address: $userInput.voucherHubAddress
+            Start Block: $userInput.voucherHubStartBlock
+            Version Label: $userInput.versionLabel
+            Deployer image version: $userInput.dockerImageVersion
+        """
+
+        if (!(userInput.dockerImageVersion?.trim())) {
+            error "docker image version parameter cannot be null or empty."
+        }
+    }
+
+    stage('Clean Workspace') {
+        cleanWs()
+    }
+
+    stage('Checkout scm') {
+        checkout scm
+    }
+
+    stage('Run operation') {
+        withCredentials([
+            usernamePassword(credentialsId: 'docker-regis', usernameVariable: 'username', passwordVariable: 'password')
+        ]) {
+            // Login to download Docker image
+            sh "echo -n '${password}' | docker login --username '${username}' --password-stdin docker-regis.iex.ec"
+
+            // Run the container for subgraph deployment
+            sh """
+                docker run \
+                    --rm \
+                    -e NETWORK_NAME='$userInput.networkName' \
+                    -e VOUCHER_HUB_ADDRESS='$userInput.voucherHubAddress' \
+                    -e VOUCHER_HUB_START_BLOCK='$userInput.voucherHubStartBlock' \
+                    -e IPFS_URL='http://$userInput.targetRemoteHost:5001' \
+                    -e GRAPHNODE_URL='http://$userInput.targetRemoteHost:8020' \
+                    -e VERSION_LABEL='$userInput.versionLabel' \
+                    docker-regis.iex.ec/voucher-subgraph-deployer:$userInput.dockerImageVersion
```

Comment:
Why do we push to `dockerhub` but for deployment we pull from `docker-regis`?

Reply:
Will be done in a Next PR